app/models/ingredients_alchemical_property.rb

```diff
@@ -81,4 +75,14 @@ def ensure_match_exists
 
     errors.add(:base, DOES_NOT_MATCH)
   end
+
+  def attributes_to_match
+    {
+      alchemical_property_id:,
+      ingredient_id: canonical_ingredients.ids,
+      strength_modifier:,
```

Comment:
Should we still match these if they are `nil`?

Reply:
Answer: no. In fact, we probably shouldn't even define these methods on the `IngredientsAlchemicalProperty` itself and should instead delegate the `strength_modifier` and `duration_modifier` methods to the `Canonical::IngredientsAlchemicalProperty` model.

Reply:
Update: I have [created a card](https://trello.com/c/Qxnqde8s/326-remove-strengthmodifier-and-durationmodifier-from-non-canonical-ingredientsalchemicalproperty-model) for this.